src/main/kotlin/no/skatteetaten/aurora/boober/utils/RetryingRestTemplateWrapper.kt

```diff
@@ -34,9 +35,10 @@ open class RetryingRestTemplateWrapper(val restTemplate: RestTemplate, open val
     fun <U : Any> get(responseType: KClass<U>, url: String, vararg uriVars: Any): ResponseEntity<U> =
         get(HttpHeaders(), responseType, url, *uriVars)
 
+    fun uri(url: String, vararg uriVars: Any) = restTemplate.uriTemplateHandler.expand(url, *uriVars)
```

Comment:
Jeg er nesten litt fristet til å si at denne burde være String.expandUri(varargs...) evt om den ligger på RestTemplate som en extention method der.

Reply:
Jeg er nok ikke helt enig her. Når vi skal akksessere denne utenfra RetryingRestTemplateWrapper så må vi kalle restTemplaten direkte for å utlede uri. eksempel:
```
val restTemplate = RetyringRestTemplateWrapper()
val uri = restTemplate.restTemplate.expandUri()
```

Hvis String brukes må restTemplate sendes med inn for a få tak i samme uriTemplateHandler